servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/catalog/ObjectStorageMockTestResourceLifecycleManager.java

```diff
@@ -37,10 +37,7 @@ public class ObjectStorageMockTestResourceLifecycleManager
   public static final String INIT_ADDRESS =
       "ObjectStorageMockTestResourceLifecycleManager.initAddress";
 
-  public static final String S3_INIT_ADDRESS =
```

Comment:
On my machine:

```
> Task :nessie-quarkus:test

1119 tests completed, 104 failed, 448 skipped
```

Reply:
In fact, just `localhost` seems to work fine on macOS. I suggest to remove this field completely, remove the `init()` method and just use `localhost` all the time, if that works on linux as well.